zk-sdk/Cargo.toml

```diff
@@ -41,5 +43,9 @@ zeroize = { workspace = true, features = ["zeroize_derive"] }
 [lib]
 crate-type = ["cdylib", "rlib"]
 
+[profile.release]
+lto = true
+opt-level = "s"
```

Comment:
Is this a separate change?

Reply:
Yeah this was put in initially as specified in the bindgen docs to minimize the wasm binary size as much as possible. It would be nice to be able to add a target specific profile in cargo, but it seems that it is not possible. 

For the changes in this PR though, adding this profile seems to have no impact on the binary size, so I removed it. For subsequent PRs, I will experiment with adding this profile to see if it affects the binary size.